### ISSUE TITLE
any chance android support could be added?

### ISSUE THREAD

**Issue by Talor-A** [apollographql/GitHub-GraphQL-API-Example#8]
Hi, I'd be willing to submit a PR to add android support to this project, would there be interest in this? Of course `NavigatorIOS` would have to be replaced with something like React-Navigation, and I don't know if there's any other platform-specific code that would need to be refactored as well.

**Comment by stubailo** [comment]
Yes please! It was iOS only because I didn't have time to worry about android as well. It would be great to add that to the example!

**Comment by Talor-A** [comment]
great, it's gonna be a pretty sizeable amount of code that will change but I'll see what I can do
